.gitlab/trigger_release/trigger_release.yml

```diff
@@ -55,3 +55,32 @@ trigger_manual_prod_release:
     - if: $CI_COMMIT_TAG =~ /^[0-9]+\.[0-9]+\.[0-9]+-v[0-9]+\.[0-9]+\.[0-9]+(-rc\.[0-9]+){0,1}$/
       when: never
     - !reference [.on_deploy_stable_or_beta_manual_auto_on_stable]
+
+include:
+  - https://gitlab-templates.ddbuild.io/slack-notifier/v3-sdm/template.yml
+
+.setup_macos_github_app_for_runner_bump:
+  # GitHub App rate-limits are per-app. Since we are rarely calling the job, we are only using the instance 2
+  - |
+    GITHUB_KEY_B64=$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $MACOS_GITHUB_APP_2 key_b64) || exit $?; export GITHUB_KEY_B64
+    GITHUB_APP_ID=$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $MACOS_GITHUB_APP_2 app_id) || exit $?; export GITHUB_APP_ID
+    GITHUB_INSTALLATION_ID=$($CI_PROJECT_DIR/tools/ci/fetch_secret.sh $MACOS_GITHUB_APP_2 installation_id) || exit $?; export GITHUB_INSTALLATION_ID
+    echo "Using GitHub App instance 2"
```

Comment:
I think it might still makes sense to reuse the existing template: https://github.com/DataDog/datadog-agent/blob/dbadf918107b26e1733db87a51801f0f79893ff8/.gitlab/common/shared.yml#L28-L42

Sure, it may be overkill to have the "load balancing" for this one, but it also doesn't hurt and it avoids duplication.

Reply:
There's also the reason that I enabled the buildenv repository only for this instance. I wanted to avoid potential risk of using too much (because of a bug) runner and preventing/slowing macOS builds

Reply:
I see!